.gitignore

```diff
@@ -19,3 +19,7 @@ lostromos
 
 # JetBrains IDE files
 .idea
+
+# Test Data used in Integration Testing
+test-data/cr_nemo_update.yml
```

Comment:
Should this really be ignored?

Reply:
Yes. That is a test created file that takes the base cr_nemo.yml, and adds a new attribute to it, then updates over the previous Nemo. It should never exist in the github repo since it should stay as a clone of nemo at runtime.